src/test/java/browserstack/utils/Log.java

```diff
@@ -0,0 +1,38 @@
+package browserstack.utils;
+
+import java.text.SimpleDateFormat;
+
+/**
+ * Logger
+ * 
+ * @author Pavel Yampolsky
```

Comment:
is there a real need for this class?

`slf4j-simple` would take care of this functionality with just a config file